core/src/main/java/org/infinispan/interceptors/impl/EntryWrappingInterceptor.java

```diff
@@ -583,9 +602,9 @@ public Object visitPutMapCommand(InvocationContext ctx, PutMapCommand command) t
          if (newMap.size() > 0) {
             PutMapCommand clonedCommand = commandFactory.buildPutMapCommand(newMap,
                   command.getMetadata(), command.getFlagsBitSet());
-            invokeNextInterceptor(ctx, clonedCommand);
+            ctx.forkInvocationSync(clonedCommand);
          }
-         return null;
+         return ctx.continueInvocation();
```

Comment:
why ```continueInvocation()```? I was expecting the ```shortCircuit()``` since we don't need to process commands in which the node is not an owner.